### ISSUE TITLE
few-shot training

### ISSUE THREAD

**Issue by wawpaopao** [clinicalml/sc-foundation-eval#2]
hello！ Nice work!
If I understand correctly , Table 1 used all-training data to train but Figure 2 use fraction of training data while Everything else remains the same? And what about using full-parameter fine-tuning. The code used linear-probing that freeze most of the weights.

**Comment by rboiarsky** [comment]
Thanks for your interest! Happy to help clarify. 

>  Table 1 used all-training data to train but Figure 2 use fraction of training data while Everything else remains the same?

that's right


> And what about using full-parameter fine-tuning. The code used linear-probing that freeze most of the weights.

We tried to follow the approach of the original authors for each model. Regarding figures 2 and 3 specifically:

In the original scBERT paper, they froze most of the weights in their encoder when doing fine-tuning, so we did the same. I doubt that unfreezing more weights would help with validation/test set performance. I would predict that it would just lead to overfitting the training data, especially in the few-shot setting. But we did not test that specifically. 

For scGPT, we did not freeze the encoder weights when fine-tuning (even in the few shot setting), as this was their setting in their cell type annotation tutorial: https://github.com/bowang-lab/scGPT/blob/main/tutorials/Tutorial_Annotation.ipynb (you can search for `freeze = False`)

If you have further questions about our choice to use un/frozen weights for other experiments/results, I would love to hear in the comments. Thanks!
